### DISCUSSION TITLE
Updating Erlang OTP and RabbitMQ server version

### DISCUSSION THREAD

**Discussion by ankushmadankar** [rabbitmq/rabbitmq-dotnet-client#1289] [Q&A] [answered]
Hello,

We are updating Erlang OTP version from 24.1.2 to 25.1 and RabbitMQ server from 3.9.7 to 3.11.2. This to address [Erlang Client Authentication Bypass Vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2022-37026).
Currently we are using RabbitMQ.Client version 6.2.2. After updating Erlang/RabbitMQ, will RabbitMQ.Client (6.2.2) break?

Thanks!

**Comment by lukebakken** [comment, marked as answer]
No, you'll be fine.
